PixiEditor/Views/MainWindow.xaml

```diff
@@ -136,6 +137,8 @@
                               IsCheckable="True" InputGestureText="Ctrl+`"/>
                     <MenuItem Header="_Open Startup Window" ToolTip="Hello there!"
                               Command="{Binding MiscSubViewModel.OpenHelloThereWindowCommand}"/>
+                    <MenuItem Header="_Open Navigation Window"
```

Comment:
I would change the "_Open Startup Window" to "Open _Startup Window" and "_Open Navigation Window" to "Open _Navigation Window". This way when you press alt two different letters will be highlighted

Reply:
TIL what that _ is for